script.service.janitor/addon.xml

```diff
@@ -0,0 +1,58 @@
+
+[I]Originally developed by Andrew 'drewzh' Higginson (xbmcfilecleaner@drewzh.com)[/I]
+        </description>
+        <summary lang="nl_NL">Verwijdert regelmatig oude films, afleveringen van TV-series en videoclips</summary>
+        <description lang="nl_NL">
+Deze addon zal automatisch de videobibliotheek scannen op zoek naar bekeken films, afleveringen van TV-series en videoclips, en deze van de harde schijf verwijderen en daarbij letten op verscheidene criteria, zoals leeftijd, cijfer, vrije harde schijfruimte, etc. Al deze opties, en andere, zijn in te stellen via het optiesmenu van de addon. Let er daarbij ook op dat je het juiste pad instelt naar de schijf die je wil gebruiken voor het controleren van de vrije schijfruimte, met name als je videocollectie op een andere harde schijf staat of op een netwerkshare. Via de instellingen kun je tevens bekijken welke video's er opgeschoond zijn.
+
+[B]Waarschuwing![/B]
+Deze addon kan in een keer je hele videocollectie verwijderen! Zorg dat je precies weet wat deze addon doet voor je voor een type video's het opschonen activeert in de opties. Ik ben [B]niet[/B] verantwoordelijk voor verlies van gegevens door handelen van de gebruiker.
+
+[I]Oorspronkelijk ontwikkeld door Andrew 'drewzh' Higginson (xbmcfilecleaner@drewzh.com)[/I]
+        </description>
+        <summary lang="fr_FR">Archive régulièrement les films, séries TV et clips musicaux</summary>
+        <description lang="fr_FR">
+        Cet add-on va scanner automatiquement  votre librairie, pour supprimer ou archiver les films, séries TV et clips musicaux basés sur de nombreux critères comme la date de visionnage, la note, l'espace disque, etc. Toutes ces options peuvent être configurées via les paramètres de l'Addon. Assurez vous de renseigner le bon chemin d'accès pour vérifier l'espace disque, spécialement lorsque vous archivez dans un second disque ou un emplacement réseau. Dans les paramètres vous pouvez accéder à l'historique des vidéos nettoyées.
+
+[B]Attention![/B]
+Cet addon à la faculté de supprimer entièrement votre collection vidéo! Assurez vous de bien comprendre à quoi est destiné cet addon avant d'activer un nettoyage dans les paramètres. Je ne pourrait aucunement être tenu responsable en cas de perte de fichier lié à une mauvaise utilisation.
+
+[I]developé par Andrew 'drewzh' Higginson (xbmcfilecleaner@drewzh.com)[/I]
+        </description>
+        <disclaimer lang="en_GB">It is recommended to not delete videos permanently until you are familiar with all of the settings.</disclaimer>
+        <disclaimer lang="nl_NL">Het is verstandig om videos pas permanent te verwijderen als je bekend bent met alle instellingen.</disclaimer>
+        <disclaimer lang="fr_FR">Il est recommandé de ne pas supprimer physiquement de vidéos avant de bien maitriser les réglages.</disclaimer>
+        <platform>all</platform>
+        <language>en nl fr</language>
```

Comment:
The language is for indication the language of the content the add-on provides. As this add-on doesn't provide content on it's own either remove the tag or set it to an empty value.
```suggestion
```